prometheus_all/resources.tf

```diff
@@ -24,11 +32,13 @@ module prometheus {
   monitoring_instance_name          = var.monitoring_instance_name
   monitoring_space_id               = data.cloudfoundry_space.monitoring.id
   paas_prometheus_exporter_endpoint = module.paas_prometheus_exporter[0].endpoint
+  redis_prometheus_exporter_endpoint= var.redis_service_instance_id == "" ? "" : module.redis_prometheus_exporter[0].endpoint
```

Comment:
Shouldn't it be
`var.redis_service_instance_id == null ? null : module.redis_prometheus_exporter[0].endpoint`

Reply:
agreed